Products/Archetypes/skins/archetypes/date_components_support.py

```diff
@@ -25,7 +25,7 @@
 # from CMFPlone.DublinCore
 CEILING = DateTime(9999, 0)
 FLOOR = DateTime(1970, 0)
-PLONE_CEILING = DateTime(2021, 0)  # 2020-12-31
+PLONE_CEILING = DateTime(2051, 0)  # 2050-12-31
```

Comment:
Something like `datetime.datetime.now().year + 30` could solve this without committing to AT in 30 years again ;-) (SCNR)